controllers/machineset_controller_test.go

```diff
@@ -125,7 +125,10 @@ var _ = Describe("MachineSet Reconciler", func() {
 
 		// Create the MachineSet.
 		Expect(k8sClient.Create(ctx, instance)).To(BeNil())
-		defer k8sClient.Delete(ctx, instance)
+		// defer func() {
```

Comment:
:grimacing: was this intentional to comment this out?

Reply:
Fixed in https://github.com/kubernetes-sigs/cluster-api/pull/1603

Reply:
no :( i'll revert this on #1599